Dockerfiles/Dockerfile

```diff
@@ -1,27 +1,22 @@
-# Build the manager binary
 ARG GOLANG_VERSION=1.19
-ARG USE_LOCAL=false
+ARG FETCH_MANIFESTS=true
```

Comment:
Does it make sense to leave it here (except as a matter of documentation) since according to the Dockerfile documentation ARG is valid till the next FROM (USE_LOCAL was expanded in the FROM line itself before)?